security/handler/security-handler-saml/src/test/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandlerTest.java

```diff
@@ -64,11 +78,39 @@ public static Document readXml(InputStream is)
     }
 
     /**
+     * Encodes the SAML assertion as a deflated Base64 String so that it can be used as a Header.
+     *
+     * @param token SAML assertion as a string
+     * @return String
+     * @throws WSSecurityException if the assertion in the token cannot be converted
+     */
+    public static String encodeSaml(String token) throws WSSecurityException {
```

Comment:
@pklinef Same reason as the decode method. I'll have this fixed with my next push.